app/components/add-preprint-box.js

```diff
@@ -1,4 +1,5 @@
-import Ember from 'ember';
+import { inject } from '@ember/service';
```

Comment:
The [recommended way](https://guides.emberjs.com/v2.18.0/applications/services/#toc_accessing-services) to do this is:
```
import { inject as service } from '@ember/service';
```